integration/integrate-with-image-vulnerability-scanners.adoc

```diff
@@ -6,13 +6,40 @@ include::modules/common-attributes.adoc[]
 toc::[]
 
 [role="_abstract"]
-{product-title} (RHACS) integrates with various vulnerability scanners to enable you to import your container images and watch them for vulnerabilities.
-You can set up {product-title} to obtain image vulnerability data from many open source and commercial container image vulnerability scanners, including:
+{rh-rhacs-first} integrates with several vulnerability scanners to enable you to import your container images and watch them for vulnerabilities.
 
+[discrete]
+== Supported container image registries
+
+Red Hat supports the following container image registries:
+
+* Amazon Elastic Container Registry (ECR)
+* Generic Docker registries (any generic Docker or Open Container Initiative-compliant image registries, for example, DockerHub, `gcr.io`, `mcr.microsoft.com`)
+* Google Container Registry
+* Google Artifact Registry
+* IBM Cloud Container Registry
+* JFrog Artifactory
+* Microsoft Azure Container Registry (ACR)
+* Red Hat Quay
+* Red Hat registry (`registry.redhat.io`, `registry.access.redhat.com`)
+
+This enhanced support gives you greater flexibility and choice to manage your container images in your preferred registry.
+
+[discrete]
+== Supported Scanners
+
+You can set up {product-title-short} to obtain image vulnerability data from the following commercial container image vulnerability scanners:
+
+* {product-title-short} Scanner (recommended)
 * link:https://github.com/quay/clair[Clair]
 * link:https://cloud.google.com/container-registry/docs/container-analysis[Google Container Analysis]
 * link:https://quay.io[Red Hat Quay]
 
+[IMPORTANT]
+====
+{product-title-short} Scanner is the most recommended and preferred among all other commercial container image vulnerability scanners. Read more about scanning container images with {product-title-short} Scanner in xref:../operating/examine-images-for-vulnerabilities.adoc#scanning-images_examine-images-for-vulnerabilities[Scanning images].
```

Comment:
```suggestion
{product-title-short} Scanner is the preferred image vulnerability scanner to use with {product-title-short}. For more information about scanning container images with {product-title-short} Scanner, see xref:../operating/examine-images-for-vulnerabilities.adoc#scanning-images_examine-images-for-vulnerabilities[Scanning images].
```

Reply:
OpenShift documentation is pretty standard in its references ("for more information, see x") so I've made that suggestion here. Also I suggested a change to the first sentence to kind of focus in on the suggestion that our scanner works best with our product (even though you can use other scanners); WDYT?

Reply:
SGTM!